brevets/flask_brevets.py

```diff
@@ -0,0 +1,124 @@
+"""
+Replacement for RUSA ACP brevet time calculator
+(see https://rusa.org/octime_acp.html)
+
+"""
+
+import flask
+from flask import request
+import arrow  # Replacement for datetime, based on moment.js
+import acp_times  # Brevet time calculations
+import os
+import logging
+import requests
+
+###
+# Globals
+###
+app = flask.Flask(__name__)
+app.debug = True if "DEBUG" not in os.environ else os.environ["DEBUG"]
+port_num = True if "PORT" not in os.environ else os.environ["PORT"]
+app.logger.setLevel(logging.DEBUG)
+
+###
+# API Callers
+###
+
+API_ADDR = os.environ["API_ADDR"]
+API_PORT = os.environ["API_PORT"]
+API_URL = f"http://{API_ADDR}:{API_PORT}/api"
+
+def brevet_insert(start_time, length, checkpoints):
+    _id = requests.post(f"{API_URL}/brevets", json={"start_time": start_time, "length": length, "checkpoints": checkpoints}).json()
+    return _id
+
+def brevet_find():
+    lists = requests.get(f"{API_URL}/brevets").json()
+    data = lists[-1]
+    return data["start_time"], data["length"], data["checkpoints"]
```

Comment:
Only issue with this is in the case that the database is empty. This will get an index error. I notice you use a try and except from where this method is called, but I might recommend another way just in case. I'm not entirely sure on what best practice is, and your idea probably makes more sense, but I thought I might as well share this 
just as another option:
```suggestion
    if lists: # Check if lists is empty or not
      data = lists[-1]
      return data["start_time"], data["length"], data["checkpoints"]
    else:
      return 0 # Indicating the status to be returned
```

If you were to implement this, then you'd need to check if what was returned was 0 or not, and then if it was return a status of 0 to the JS with the corresponding message. Otherwise, you could then get your variables "start_time, length, checkpoints" and set them equal to the return value. 

Once again, this kind of adds more code and the try and except is probably easier - I'm not sure what best practice is here.